src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -117,6 +119,25 @@ public virtual string CreateToken(string payload, SigningCredentials signingCred
             return CreateTokenPrivate(JObject.Parse(payload), signingCredentials, null, null);
         }
 
+        /// <summary>
+        /// Creates a JWS(Json Web Signature).
+        /// </summary>
+        /// <param name="tokenDescriptor">A <see cref="SecurityTokenDescriptor"/> that contains details of contents of the token.</param>
+        /// <returns>A JWS in Compact Serialization Format.</returns>
+        public virtual string CreateToken(SecurityTokenDescriptor tokenDescriptor)
+        {
+            if (tokenDescriptor == null)
```

Comment:
Can we added tests for SecurityTokenDescriptor properties.

Reply:
Done!